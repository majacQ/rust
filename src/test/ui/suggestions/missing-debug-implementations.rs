#![allow(unused)]
#![deny(missing_debug_implementations)]

struct Pod;

pub struct Discovery {}

pub struct Shenzhou {
    escape_pods: [Pod]
}

fn main() {}
