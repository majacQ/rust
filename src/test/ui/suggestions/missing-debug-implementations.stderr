error: type does not implement `fmt::Debug`
 --> $DIR/missing-debug-implementations.rs:6:1
  |
6 | pub struct Discovery {}
  | --------------------^^^
  | |
  | help: try deriving `Debug`: `#[derive(Debug)] pub struct Discovery`
  |
note: lint level defined here
 --> $DIR/missing-debug-implementations.rs:2:9
  |
2 | #![deny(missing_debug_implementations)]
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: type does not implement `fmt::Debug`
  --> $DIR/missing-debug-implementations.rs:8:1
   |
8  | / pub struct Shenzhou {
9  | |     escape_pods: [Pod]
10 | | }
   | |_^

error: aborting due to 2 previous errors

